Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/warnothers/SubmissionResultPositiveOtherWarningNoConsentViewModel.kt

```diff
@@ -37,7 +37,8 @@ class SubmissionResultPositiveOtherWarningNoConsentViewModel @AssistedInject con
     private val coronaTestProvider: CoronaTestProvider,
     private val checkInRepository: CheckInRepository,
     private val analyticsKeySubmissionCollector: AnalyticsKeySubmissionCollector,
-    @Assisted private val testIdentifier: TestIdentifier
+    @Assisted private val testIdentifier: TestIdentifier,
+    @Assisted private val comesFromDispatcherFragment: Boolean
```

Comment:
Just add @Suppress("LongParameterList") above class, its frequently used in codebase for exactly this problem

Reply:
Thanks <3